mapie/control_risk/ltt.py

```diff
@@ -0,0 +1,120 @@
+import warnings
+
+from typing import Tuple, List, Optional, Any
+from mapie._typing import NDArray, ArrayLike
+
+import numpy as np
+
+from .p_values import compute_hoefdding_bentkus_p_value
+
+
+def _ltt_procedure(
+    r_hat: NDArray,
+    alpha_np: NDArray,
+    delta: Optional[float],
+    n_obs: int
+) -> Tuple[List[List[Any]], NDArray]:
+    """
+    Apply the Learn-Then-Test procedure for risk control.
+    This procedure is called in multi-label-classification
+    if ``metric=precision``.
```

Comment:
```suggestion
    This procedure is called in ``MapieMultiLabelClassifier``
    when ``metric_control=precision`` and ``method=ltt``.
```

Reply:
DONE

Reply:
@SZiane, Are you sure it's done? I can't see the change.

Reply:
Now its done